.github/CODEOWNERS

```diff
@@ -6,3 +6,7 @@
 
 # Katamari changes should include the integrations team as well
 /modules/katamari/ @tinymce/tinymce-reviewers @tinymce/integrations-team
+
+# Changelogs should include docs team
+/.changes/ @tinymce/tinymce-reviewers @tinymce/documentation-team
```

Comment:
Won't this pick up the yml files we just want docs to approve the merged `.md` files. Otherwise docs will be included on all PRs?

Reply:
We will ignore the `unreleased` and other folders and just focus on changes to `tinymce` then. As originally suggested.